google/cloud/aiplatform/training_jobs.py

```diff
@@ -1013,7 +1015,8 @@ class _CustomTrainingJob(_TrainingJob):
 
     def __init__(
         self,
-        display_name: str,
+        # TODO: Make display_name parameter fully optional in next major release
```

Comment:
Please create and add a buganizer ticket.

Reply:
Done.